pkg/server/sandbox_run.go

```diff
@@ -48,6 +48,32 @@ func init() {
 		"github.com/containerd/cri-containerd/pkg/store/sandbox", "Metadata")
 }
 
+// privilegedSandbox returns true if the sandbox configuration
+// requires additional host privileges for the sandbox.
+func (c *criContainerdService) privilegedSandbox(req *runtime.RunPodSandboxRequest) bool {
```

Comment:
We need a simple unit test for this to make sure the behavior is correct, and no future change will break this.
And I don't think this is necessarily need to be a member function of `criContainerdService`, it is not used. :)

Reply:
Done :)